### ISSUE TITLE
[Bug] Incorrect dimension index in loadCam prevents Alpha Mask from being loaded

### ISSUE THREAD

**Issue by superzheren** [fastgs/FastGS#24]
### **Issue Description:**

**Describe the bug**
The current implementation of Alpha Mask loading in `utils/camera_utils.py` uses an incorrect dimension index to check for the number of channels. It checks `resized_image_rgb.shape[1] == 4`, where `shape[1]` corresponds to the **height** of the image in PyTorch's `(C, H, W)` format, rather than the **number of channels**. As a result, the mask loading logic is never triggered unless the image height is exactly 4 pixels.

**To Reproduce**
1. Prepare a dataset with 4-channel RGBA PNG images (with transparency).
2. Run training with any script.
3. Observe that the background (Alpha = 0 area) is still being trained/rendered, or add a print statement in `loadCam` to check if `loaded_mask` is ever assigned.

**Code Snippet causing the issue:**
In `utils/camera_utils.py`, line 39 (approx.):

```python
# Current incorrect code
if resized_image_rgb.shape[1] == 4:
    loaded_mask = resized_image_rgb[3:4, ...]
```

**Expected behavior**
The code should check `shape[0]` for the channel count, as `PILtoTorch` returns a tensor in `(Channels, Height, Width)` format.

**Screenshots/Log (Optional)**
If debugging, the tensor shape typically looks like: `torch.Size([4, 1600, 1600])`.
- `shape[0]` is 4 (RGBA).
- `shape[1]` is 1600 (Height).

**Proposed Fix**
Change the index from `1` to `0`:

```python
# Fixed code
if resized_image_rgb.shape[0] == 4:
    loaded_mask = resized_image_rgb[3:4, ...]
```

**Environment:**
- FastGS Version: Latest Main
- OS: Linux/Ubuntu (WSL)
- PyTorch Version: 1.12.1 (Standard for FastGS)
